eng/Directory.Build.Data.props

```diff
@@ -75,8 +75,8 @@
 
   <PropertyGroup Condition="'$(IsTestProject)' == 'true' or '$(IsTestSupportProject)' == 'true' or '$(IsSamplesProject)' == 'true'">
     <IsPackable>false</IsPackable>
-    <RequiredTargetFrameworks>netcoreapp2.1</RequiredTargetFrameworks>
-    <RequiredTargetFrameworks Condition="'$(OS)' == 'Windows_NT'">netcoreapp2.1;net461</RequiredTargetFrameworks>
+    <RequiredTargetFrameworks>netcoreapp2.1;netcoreapp3.1</RequiredTargetFrameworks>
```

Comment:
The build must be failing because of either this change or the DotNetCoreSDKVersion change. I'd suggest trying to isolate each to see what is causing the build failures.